### DISCUSSION TITLE
Extension detected by FoE

### DISCUSSION THREAD

**Discussion by jjdewit** [cosmos0nash/FoE-Extender#26] [General]
Hi all, 

Last night I received a (probably) automatic message from the FoE team about using bots/scripts:
```
Our system has detected that you are using prohibited software to interact with our game.
Please note that using bots, scripts or other tools that give you an unfair advantage is strictly prohibited in our game. We ask that you immediately disable the prohibited software, otherwise additional sanctions may follow.
Thank you for your cooperation.

Your Forge of Empires team
```

The FoE-Extender extension claims to be undetectable. 

Should I cancel my subscription and delete the extension or not? I don't want to lose my account.

**Comment by cosmos0nash** [comment]
Have you ever tried to read the help on the site instead of asking such questions? It's all written there just for you
